Update review doc (#1350)
avohq/docs: pages/workflow/review.mdx

---

> title: '2. Review changes'

Reviewer: Need to represent this in the nav as well

> ## Submit a review

Reviewer: ```suggestion
## Submitting a review
```

The role of the reviewer(s) is to make sure that the changes

- Fit your data needs
- Match the data design standards and conventions of your organization
- Don’t negatively impact other teams using the data you are changing
- Are feasible for implementation

> The branch review screen in Avo summarizes the changes that have been suggested on the branch.

Reviewer: ```suggestion
The "view changes" screen in Avo summarizes the changes that have been suggested on the branch.
```

> From there, you can comment on every item that has been introduced or changed on your branch, ask questions and suggest changes.

Reviewer: ```suggestion
###Commenting on changes

From the "view changes screen", you can comment on every item that has been introduced or changed on your branch, ask questions and suggest changes.
```
